### DISCUSSION TITLE
Cask and regular install

### DISCUSSION THREAD

**Discussion by lavinnyv** [Homebrew/discussions#5128] [Casks]
### Output of `brew config`

```shell
this is a more general question about cask vs regular install
```

### Output of `brew doctor`

```shell
this is a more general question about cask vs regular install
```

### Description of issue

i normally just install mpv --cask. should i be installing mpv and mpv --cask? i haven't in the past and when i just ran brew info mpv it says not installed, but i do get mpv to work and have a mpv.app in my applications folder

this is what i get:
$ brew info mpv
Warning: Treating mpv as a formula. For the cask, use homebrew/cask/mpv
==> mpv: stable 0.37.0, HEAD
Media player based on MPlayer and mplayer2
https://mpv.io
Not installed

it says the version number, but also says not installed a few lines down

### Relevant casks

mpv

**Comment by SMillerDev** [comment]
> Warning: Treating mpv as a formula. For the cask, use homebrew/cask/mpv

So you have the cask installed and if you just want the .app that's fine.

**Comment by lavinnyv** [comment]
i use it cli as well, and never ran into a problem. but i was running the brew info on mpv and there was the line that said not installed. is it fine/better to run brew install mpv && brew install mpv --cask?

**Reply by gromgit** [reply]
> i was running the brew info on mpv and there was the line that said not installed.

Which is correct, because you have the cask installed, not the formula.

> is it fine/better to run brew install mpv && brew install mpv --cask?

No. If you tried that, you should get an error, because:
```bash
$ brew cat --cask mpv | grep conflicts_with
  conflicts_with formula: "mpv"
```
and this line exists because the cask symlinks the cask's `mpv` binary into the Homebrew `bin` directory:
```bash
$ brew cat --cask mpv | grep binary
  binary "#{appdir}/mpv.app/Contents/MacOS/mpv"
```
You can either install the formula or the cask, but not both.

**Comment by lavinnyv** [comment]
great. thanks for the help!
